src/HubphAPI.php

```diff
@@ -73,15 +73,20 @@ public function prGet($org, $project, $id)
         return $this->gitHubAPI()->api('pull_request')->show($org, $project, $id);
     }
 
-    public function prCreate($org, $project, $title, $body, $base, $head, &$response = null)
+    public function prOpen($org, $project, $title, $body, $base, $head)
     {
         $params = [
             'title' => $title,
             'body' => $body,
             'base' => $base,
             'head' => $head,
         ];
-        $response = $this->gitHubAPI()->api('pull_request')->create($org, $project, $params);
+        return $this->gitHubAPI()->api('pull_request')->create($org, $project, $params);
+    }
+
+    public function prCreate($org, $project, $title, $body, $base, $head)
+    {
+        $this->prOpen($org, $project, $title, $body, $base, $head);
```

Comment:
`$response = `

Or move log up to prOpen

Reply:
whoops! Captured